kimchi/src/circuits/polynomials/foreign_mul/mod.rs

```diff
@@ -0,0 +1,104 @@
+//! Foreign field multiplication gate
```

Comment:
As currently implemented, this collection of gates is more of an 88-bit range-check gate than anything to do with foreign field arithmetic. Worth renaming?

Reply:
Done!